server/app/controllers/admin/AdminProgramController.java

```diff
@@ -78,9 +79,26 @@ public Result index(Request request) {
     Optional<CiviFormProfile> profileMaybe = profileUtils.currentUserProfile(request);
     return ok(
         listView.render(
-            programService.getActiveAndDraftProgramsWithoutQuestionLoad(),
+            programService.getInUseActiveAndDraftProgramsWithoutQuestionLoad(),
             questionService.getReadOnlyQuestionServiceSync(),
             request,
+            ProgramTab.IN_USE.getProgramTab(),
```

Comment:
suggestion (maybe blocking): Instead of calling `getProgramTab` and passing along a string, keep it as the enum until the point where you need to it to be a string (I think that's when you call `.setText(status)`. At that point, you can do `.setText(status.getProgramTab()` instead)`. That way we can have greater control over what's allowed to be passed as a value.

Reply:
+1 This is sometimes (pejoratively) called being "stringly typed" as opposed to "strongly typed".

Reply:
that makes sense, it also makes it less error-prone. change applied.